pr_agent/tools/pr_description.py

```diff
@@ -203,6 +204,16 @@ def _prepare_labels(self) -> List[str]:
                 pr_types = self.data['type']
             elif type(self.data['type']) == str:
                 pr_types = self.data['type'].split(',')
+
+        # convert lowercase labels to original case
+        try:
+            if "labels_minimal_to_labels_dict" in self.variables:
+                d: dict = self.variables["labels_minimal_to_labels_dict"]
+                for i, label_i in enumerate(pr_types):
+                    if label_i in d:
+                        pr_types[i] = d[label_i]
+        except Exception as e:
+            get_logger().error(f"Error converting labels to original case {self.pr_id}: {e}")
```

Comment:
**Suggestion:** Use list comprehension to convert lowercase labels to original case.
```suggestion
        try:
            if "labels_minimal_to_labels_dict" in self.variables:
                d: dict = self.variables["labels_minimal_to_labels_dict"]
                pr_types = [d[label_i] if label_i in d else label_i for label_i in pr_types]
        except Exception as e:
            get_logger().error(f"Error converting labels to original case {self.pr_id}: {e}")
```